packages/gatsby-remark-code-repls/src/constants.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const { join } = require(`path`)
+
+module.exports = {
+  OPTION_DEFAULT_CODEPEN_EXTERNALS: [
+    `https://unpkg.com/react/umd/react.development.js`,
+    `https://unpkg.com/react-dom/umd/react-dom.development.js`,
+  ],
+  OPTION_DEFAULT_DIRECTORY: `examples/`,
+  OPTION_DEFAULT_LINK_TEXT: `REPL`,
+  OPTION_DEFAULT_REDIRECT_TEMPLATE_PATH: join(__dirname, `default-redirect-template.js`),
+  PROTOCOL_BABEL: `babel://`,
+  PROTOCOL_CODEPEN: `codepen://`,
```

Comment:
to be in line with other parsers (browser, node) I'd omit the slashes from the protocol: `codepen:`

Reply:
You were the one who suggested I add them in the first place. Why now suggest removing them?

Reply:
You were the one who suggested I add them in the first place. Why now suggest removing them?

Reply:
Ah, you're suggesting I remove them from the constant I think. That's pretty nit picky :grin: I think it's okay as--soft?

Reply:
:D no worries, yes super nit of course :)

Reply:
lol 😆 no worries